varcode/transcript_mutation_effects.py

```diff
@@ -313,7 +315,7 @@ def get_class(effect):
             return _MultipleSubstitution
     return effect.__class__
 
-variant_effect_priority_list = [
+transcript_effect_priority_list = [
```

Comment:
One possibility to make this a *little* less hacky is to use  `Substitution` to mean single amino acid changes, and use `ComplexSubstitution` or `MultiSubstitution` for multiple amino acid changes. 

The hierarchy would then be:

`Substitution` < `Insertion` < `Deletion` < `MultipleSubstitution`